core/frontend/src/tile/map/MapLayerTileTreeReference.ts

```diff
@@ -13,22 +13,42 @@ import { IModelApp } from "../../IModelApp";
 import { IModelConnection } from "../../IModelConnection";
 import { createModelMapLayerTileTreeReference, MapLayerImageryProvider, TileTreeReference } from "../internal";
 
-/** @internal  */
+/**
+ * A reference to a [[TileTree]] to be used for map layer tiles.
```

Comment:
The description is quite thin here.  

I feel like the concept of TileTree/TileTreeReference is not documented enough in general.  We should at least reference the [TileTreeReference ](https://www.itwinjs.org/reference/core-frontend/tiles/tiletreereference/) documentation.

One key concept concept to mentions is that multiple MapLayerTileTreeReference can refer to same TileTree if the layer settings are "equivalent".